manual.asc

```diff
@@ -7,10 +7,10 @@
 
 == Introduction
 
-Thank you for using AnkiDroid, the Android client for the popular 
-http://ankisrs.net/[Anki] spaced repetition system. 
```

Comment:
There's a lot of additional spacing changes in this diff.

Could you remove these and only include relevant content changes?

Reply:
ok I will change the commit message